packages/run-protocol/src/vaultFactory/params.js

```diff
@@ -18,63 +18,107 @@ export const LIQUIDATION_MARGIN_KEY = 'LiquidationMargin';
 export const INTEREST_RATE_KEY = 'InterestRate';
 export const LOAN_FEE_KEY = 'LoanFee';
 
-/** @type {MakeElectorateParams} */
+/**
+ * @param {Amount} electorateInvitationAmount
+ */
```

Comment:
again, please consider a more dense/concise form:

```suggestion
/** @param {Amount} electorateInvitationAmount */
```

Reply:
I'm convinced. 

As you said above, there's a trade-off with comment churn when another param is added. But on balance I think it's better to have the concise form. Comments have less need for `git blame`.